netplay/createroom.cpp

```diff
@@ -158,6 +171,9 @@ void CreateRoom::onConnected()
     json.insert("client_sha", QStringLiteral(GUI_VERSION));
     json.insert("netplay_version", NETPLAY_VER);
     json.insert("lle", w->getSettings()->value("LLE").toInt() ? "Yes" : "No");
+    json.insert("use_input_delay", useInputDelay->isChecked());
```

Comment:
Is ```use_input_delay``` used anywhere on the server side? It doesn't seem like it's needed. The server just seems to check for ```json.contains("input_delay")``` which should be covered below

Reply:
Sorry I hadn't read low enough, I see that it's used in join_room, you can ignore this

Reply:
Yeah it's only client-side validation right now, that all players have input delay. I think that that's fine, at least to start, but we could maybe add server-side validation in the future.